tensorcircuit/templates/blocks.py

```diff
@@ -149,3 +149,54 @@ def example_block(
         for i in range(n):
             c.rx(i, theta=param[2 * j + 1, i])
     return c
+
+
+def qft(
+    c: Circuit,
+    *index: int,
+    do_swaps: bool = False,
```

Comment:
this should default to True, as it is part of the QFT

Reply:
Sure